README.md

```diff
@@ -28,23 +28,18 @@ The consumer is written in Rails. Ensure to have the following components instal
 
 After cloning the repo, install the necessary dependencies with `bundle exec bundle install`.
 
-Afterwards you may start the webserver with `rails s -b 0.0.0.0` where the ENV variables PRICEWARS_MARKETPLACE_URL and PRICEWARS_PRODUCER_URL point to the actual path of the marketplace and the producer.
+Afterwards you may start the webserver with `rails s -b 0.0.0.0` where the ENV variable PRICEWARS_MARKETPLACE_URL should point to the marketplace server.
 
 If all worked out, see the results at http://localhost:3000
 
 ## Configuration
 
-Currently, two ENV params are needed:
-* PRICEWARS_PRODUCER_URL
-* PRICEWARS_MARKETPLACE_URL
-
-If they are not set, the deployed VM urls are automatically used.
-
-One may simple export them to the related system:
+The marketplace url is configured in the environment variable `PRICEWARS_MARKETPLACE_URL`.
+If it is not set, a default url to a deployed VM is used.
```

Comment:
The default is the docker-internal path (`merchant`) to the marketplace, right?
Or what deployed VM do you mean?

Reply:
The default url was `vm-mpws2016hp1-04.eaalab.hpi.uni-potsdam.de:8080/marketplace`. I changed that to `http://marketplace:8080`.